pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotClientRequest.java

```diff
@@ -63,11 +63,12 @@
   @Inject
   private BrokerMetrics brokerMetrics;
 
+  @Deprecated
```

Comment:
can you add comments on deprecation reason and the alternative?

Reply:
done